packages/server/test/support/helpers/performance.js

```diff
@@ -0,0 +1,46 @@
+const _ = require('lodash')
+const execa = require('execa')
+const Promise = require('bluebird')
+const rp = require('request-promise')
+
+const API_URL = process.env.PERF_API_URL || 'http://localhost:2999/track'
+const API_KEY = process.env.PERF_API_KEY
+
+// Store this performance record permanently.
+function track (type, data) {
+  if (!API_KEY) {
+    return Promise.resolve()
+  }
+
+  return Promise.join(
+    execa.stdout('git', ['rev-parse', 'HEAD']),
+    execa.stdout('git', ['log', 'HEAD', '-1', '|', 'tail', '-n1'], { shell: true }).then(_.trim),
+    execa.stdout('git', ['branch', '|', 'grep', '\\*', '|', 'cut', '-d', '\'', '\'', '-f2'], { shell: true }),
+    (commitSha, commitMessage, branch) => {
+      const body = {
+        type,
+        data: {
```

Comment:
- Add `package.json` version of cypress here 
- Add the `NEXT_DEV_VERSION` env var, or whatever it is - so we can track against the various versions of cypress as they are built.
- Add machine info - memory, cpu, etc
- Also we may want to include the circle ci build info - so we can generate links straight to this build / job. Possibly we could just include the entire URL so we don't have to reform it later 
- `camelCase` all the properties in the data payload
- instead of using `execa.stdout` you can make use of the `ci_providers` file, which can automatically extract all the build information from environment variables, and is built to additionally fall back to parsing the `git` information. this would allow us to capture which CI environment it ran in. You could also look at some of the information we capture during recordings to see what else may be valuable - such as the commit author, etc.

Reply:
Nice, made these changes.

I don't think we need to camelCase the keys - since this is primarily for Metabase where the keys would need to be re-labeled to be human-readable anyways, plus, Mongo has no issues supporting human-readable keys.